pkg/controller/pxc/users.go

```diff
@@ -733,6 +944,20 @@ func (r *ReconcilePerconaXtraDBCluster) handleProxyadminUser(cr *api.PerconaXtra
 	}
 	logger.Info(fmt.Sprintf("User %s: internal secrets updated", user.Name))
 
+	passPropagated, err := r.isPassPropagated(cr, user)
+	if err != nil {
+		return errors.Wrap(err, "is pass propagated")
+	}
+	if !passPropagated {
+		return PassNotPropagatedError
+	}
+
+	err = r.discardOldPassword(cr, secrets, internalSecrets, user)
+	if err != nil {
+		return errors.Wrap(err, "discard proxyadmin old pass")
+	}
```

Comment:
AFAIK we don't store proxyadmin user in DB, how can we use dual passwords with it?

Reply:
RIght, fixed.